style.css

```diff
@@ -127,4 +141,50 @@ label {
 .desc {
     color: hsl(184, 14%, 56%);
     font-size: 12px;
-}
\ No newline at end of file
+}
+
+#custom {
+    background-color: hsl(189, 41%, 97%);
+    color: hsl(184, 14%, 56%);
+}
+.inactive {
+    background-color: #0b676d;
+    color: #065f63;
+}
+.active {
+    background-color: hsl(172, 67%, 45%);
+    color: hsl(183, 100%, 15%);
+}
+
+@media screen and (min-width: 600px) {
+    .tip-container {
+        width: 400px;
+    }
+}
+
+@media screen and (min-width: 800px) {
+    .bill {
+      /* Apply some styles */
+      border-radius: 20px;
+      width: 800px;
+      height: 400px;
+      display: flex;
+      justify-content: space-between;
+      flex-direction: row;
+    }
+    .tip-container {
+        height: 80%;
+        width: 45%;
+        justify-content: space-between;
+    }
+    .left-container {
+        width: 45%;
+        display: flex;
+        flex-direction: column;
+        height: 95%;
+        justify-content: space-between;
+    }
+    button {
+        width: 100px;
+    }
```

Comment:
This applies CSS to all buttons 
Not a bad practice but it is recommended to have a generic CSS that can be applied to every button throughout the project